code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -582,3 +587,123 @@
 /obj/item/clothing/head/helmet/changeling/Initialize(mapload)
 	. = ..()
 	ADD_TRAIT(src, TRAIT_NODROP, CHANGELING_TRAIT)
+
+/datum/action/changeling/suit/hive_head
+	name = "Hive Head"
+	desc = "We coat our head in a waxy outing coating similar to a bee hive which can be used to manufacture bees to attack our enemies. Costs 15 chemicals."
+	helptext = "While the hive head does not provide much in the ways of armor, it does allow the user to send bees out to attack targets. Reagents can poured inside the hive to cause all bees released to inject said reagents."
+	button_icon_state = "hive_head"
+	chemical_cost = 15
+	dna_cost = 2
+	req_human = FALSE
+	blood_on_castoff = TRUE
+
+	helmet_type = /obj/item/clothing/head/helmet/changeling_hivehead
+	helmet_name_simple = "hive head"
+
+/obj/item/clothing/head/helmet/changeling_hivehead
+	name = "hive head"
+	desc = "A strange, waxy outer coating covering your head. Gives you tinnitus."
+	icon_state = "hivehead"
+	inhand_icon_state = null
+	flash_protect = FLASH_PROTECTION_FLASH
+	item_flags = DROPDEL
+	armor_type = /datum/armor/changeling_hivehead
+	flags_inv = HIDEEARS|HIDEHAIR|HIDEEYES|HIDEFACIALHAIR|HIDEFACE|HIDESNOUT
+	actions_types = list(/datum/action/cooldown/hivehead_spawn_minions)
+	///Does this hive head hold reagents?
+	var/holds_reagents = TRUE
+
+/obj/item/clothing/head/helmet/changeling_hivehead/Initialize(mapload)
+	. = ..()
+	if(holds_reagents)
+		create_reagents(50, REFILLABLE)
+
+/datum/armor/changeling_hivehead
+	melee = 10
+	bullet = 10
+	laser = 10
+	energy = 10
+	bio = 50
+
+/obj/item/clothing/head/helmet/changeling_hivehead/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, CHANGELING_TRAIT)
+
+/obj/item/clothing/head/helmet/changeling_hivehead/attackby(obj/item/attacking_item, mob/user, params)
+	. = ..()
+	if(istype(attacking_item, /obj/item/organ/internal/monster_core/regenerative_core/legion) && holds_reagents)
```

Comment:
Invert this for an early return, doesn't seem like we do anything in any other case.